pkg/asset/installconfig/azure/validation.go

```diff
@@ -139,6 +138,11 @@ func ValidateInstanceType(client API, fieldPath *field.Path, region, instanceTyp
 			}
 		} else if strings.EqualFold(*capability.Name, "UltraSSDAvailable") {
 			ultraSSDAvailable = strings.EqualFold(*capability.Value, "True")
+		} else if strings.EqualFold(*capability.Name, string(aztypes.AcceleratedNetworkingCapabilityEnabled)) {
+			if strings.EqualFold(acceleratedNetworking, string(aztypes.AcceleratedNetworkingAccelerated)) && !strings.EqualFold(*capability.Value, "True") {
```

Comment:
There is no need to do an `EqualFold` comparison for the `acceleratedNetworking` value.
```suggestion
			if acceleratedNetworking == string(aztypes.AcceleratedNetworkingAccelerated) && !strings.EqualFold(*capability.Value, "True") {
```